### ISSUE TITLE
sanic-auth no longer works with recent sanic/sanic_session

### ISSUE THREAD

**Issue by nhi-vanye** [pyx/sanic-auth#11]
Not sure when this broke, but with sanic 20.3 and sanic_session 0.7 I'm getting

```
2020-05-30T17:40:47.800 ERROR handlers.py#140 Exception occurred while handling uri: 'http://192.168.20.55/'
Traceback (most recent call last):
  File "/usr/lib/python3.8/site-packages/sanic/app.py", line 973, in handle_request
    response = await response
  File "/usr/lib/python3.8/site-packages/sanic_auth/__init__.py", line 93, in privileged
    user = self.current_user(request)
  File "/usr/lib/python3.8/site-packages/sanic_auth/__init__.py", line 61, in current_user
    token = self.get_session(request).get(self.auth_session_key, None)
  File "/usr/lib/python3.8/site-packages/sanic_auth/__init__.py", line 143, in get_session
    return request[self.session_name]
TypeError: 'Request' object is not subscriptable
```

**Comment by matsurih** [comment]
I don't know if this repo is alive, but I met same error and I was able to avoid this problem.
You can use `request.ctx.session` instead of `request[self.session_name]`.
If you want to use sanic-auth with sanic 20.3 or later, see my solution: https://github.com/matsurih/sanic-auth/commit/907519aba32600df28531f4e073dca971f2fad9c

**Comment by pyx** [comment]
Sorry for the hiatus.
It seems like so, @matsurih , can you send a pull request, I will make a new release based on that.
Thank you.

**Comment by pyx** [comment]
Fixed by PR #12
